File: Javascript/CheckIfPrime.js
function is_prime(num){
    if(num <= 1){
        return false
    }
    for(let i = 2; i < Math.floor(Math.sqrt(num)) + 1 ; i++){
        if(parseInt(num % i) === 0 ){
            return false
        }

    }
    return true
}


for(var i = 1; i < 20; i++){
    if(is_prime(i)){
        console.log(i + ' is Prime')
    }else{
        console.log(i + ' is Non-Prime')
    }
}

File: CheckIfPrime.py
from math import sqrt

def is_prime(num):
    if num <= 1: return False

    for n in range(2,int(sqrt(num)) + 1):
        if num % n == 0:
            return False
        else: continue
    else:
        return True



# from math import floor as f
# from math import sqrt

# def is_prime(num):
#     if num <= 1: return False
#     root = int(sqrt(num))

#     for n in range(2,root + 1):
#         if num % n == 0:
#             return False
#         else: continue
#     else:
#         return True


